resources/language/resource.language.hu_hu/strings.po

```diff
@@ -273,35 +273,35 @@ msgstr "Megpróbáljuk újra?"
 ### INFORMATION DIALOG
 msgctxt "#30800"
 msgid "[B]Kodi[/B] version [B]{version}[/B] is running on a [B]{system}[/B] system with [B]{arch}[/B] architecture."
-msgstr ""
+msgstr "[B]Kodi[/B] verzió [B]{version}[/B] fut a [B]{system}[/B] rendszeren [B]{arch}[/B] architecturával."
 
 msgctxt "#30810"
 msgid "[B]InputStream Helper[/B] is at version [B]{version}[/B] {state}"
-msgstr ""
+msgstr "[B]InputStream Helper[/B] verzió [B]{version}[/B] {state}"
 
 msgctxt "#30811"
 msgid "[B]InputStream Adaptive[/B] is at version [B]{version}[/B] {state}"
-msgstr ""
+msgstr "[B]InputStream Adaptive[/B] verzió [B]{version}[/B] {state}"
 
 msgctxt "#30820"
 msgid "[B]Widevine CDM[/B] is [B]built into Android[/B]"
-msgstr ""
+msgstr "[B]Widevine CDM[/B] az [B]Androidba épülve[/B]"
 
 msgctxt "#30821"
 msgid "[B]Widevine CDM[/B] is at version [B]{version}[/B] and was installed on [B]{date}[/B]"
-msgstr ""
+msgstr "[B]Widevine CDM[/B] verzió [B]{version}[/B] telepítve [B]{date}[/B]"
 
 msgctxt "#30822"
 msgid "It was extracted from Chrome OS image [B]{name}[/B] with version [B]{version}[/B]"
-msgstr ""
+msgstr "Legutóbb kicsomagolva a Chrome OS image-ból [B]{name}[/B] verzió [B]{version}[/B]"k
```

Comment:
I think the k at the end should be removed?

Reply:
I saw it after merging. Will fix it.

Reply:
Yes. sorry, updated again